lib/ts/types.ts

```diff
@@ -39,6 +40,13 @@ export type InputType = {
     enableDebugLogs?: boolean;
     apiDomain: string;
     apiBasePath?: string;
+    /**
+     * Rather than using browser-tabs-lock, allow for a Lock factory to be configured
+     * via SuperTokens.init(...). This can be used for example by a WebExtension that
+     * needs to update cookies for a domain that may or may not have an associated
+     * tab open.
```

Comment:
```suggestion
     * This allows for a Lock factory to be configured, which defaults to browser-tabs-lock.
     * This can be used, for example, by a WebExtension that needs to update cookies for
     * a domain that may or may not have an associated tab open.
```